manage.py

```diff
@@ -128,159 +131,176 @@ def halt(self):
-        cwd=self.config.current_dir).wait()
+          ['vagrant', 'ssh', '-c', self.vagrant_ssh_cmd.format(
+              'swarm-testbed' if is_testbed else 'swarm', 'configure')
+          ],
+          cwd=self.config.current_dir).wait()
       return
-      
+
     self.use_proper_hosts(is_testbed)
     temp_env = os.environ.copy()
     temp_env['ANSIBLE_CONFIG'] = 'appdev.cfg'
     subprocess.Popen(
-      ['ansible-playbook', 'configure-stack.yml'], 
-      cwd=self.config.build_dir, 
-      env=temp_env).wait()
+        ['ansible-playbook', 'configure-stack.yml'],
+        cwd=self.config.build_dir,
+        env=temp_env).wait()
 
   def clean(self, is_testbed):
     if os.name == 'nt':
       subprocess.Popen(
-        ['vagrant', 'ssh', '-c', self.vagrant_ssh_cmd.format('swarm-testbed' if is_testbed else 'swarm', 'clean')], 
-        cwd=self.config.current_dir).wait()
+          ['vagrant', 'ssh', '-c', self.vagrant_ssh_cmd.format(
+              'swarm-testbed' if is_testbed else 'swarm', 'clean')
+          ],
+          cwd=self.config.current_dir).wait()
       return
-      
+
     self.use_proper_hosts(is_testbed)
     temp_env = os.environ.copy()
     temp_env['ANSIBLE_CONFIG'] = 'appdev.cfg'
     subprocess.Popen(
-      ['ansible-playbook', 'stack-down.yml'], 
-      cwd=self.config.build_dir, 
-      env=temp_env).wait()
+        ['ansible-playbook', 'stack-down.yml'],
+        cwd=self.config.build_dir,
+        env=temp_env).wait()
 
 def usage():
-  print("Swarm CLI tool\n")
-  print("Use this tool to setup, test, work with, and deploy to a Docker Swarm cluster.\n")
-  print("REFERENCE")
-  print("python manage.py compile BUNDLE_DIR - compile the testbed and swarm tool using the bundle at path BUNDLE_DIR")
-  print("python manage.py config - print out the complete configuration described by the swarm.ini file and the curent compilation")
-  print("")
-  print("python manage.py testbed up - boot up all nodes in the Vagrant testbed")
-  print("python manage.py testbed halt - power down all nodes in the Vagrant testbed")
-  print("python manage.py testbed destroy - destroy all nodes in the Vagrant testbed")  
-  print("")
-  print("****The following swarm operations affect PRODUCTION machines****")
-  print("python manage.py swarm lockdown - lock down access to the target production machines")  
-  print("python manage.py swarm join - join the target machines into a Docker Swarm")
-  print("python manage.py swarm configure - upload the Docker Compose file to the first manager of the Swarm")  
-  print("python manage.py swarm clean - remove Docker Stack, containers, and images from all machines")    
-  print("")
-  print("****The following swarm operations affect TESTBED machines****")
-  print("python manage.py swarm-testbed [above command] - applies swarm operation [above command] onto the testbed")    
-  
+  print("""

```

Comment:
prefer single quotes

Reply:
you can't have multi-line strings with single quotes

Reply:
`'''<multiline>'''`